### ISSUE TITLE
how to modify right and left delimiters 

### ISSUE THREAD

**Issue by huyongde** [valyala/quicktemplate#37]
{%  %} are default delimiters,  how to modify the default right and left delimiters, thanks

**Comment by dommmel** [comment]
Can't be changed easily from looking at the code. Are there plans to make them configurable? My use case is to generate liquid templates which use the same delimiters

**Comment by b3b00** [comment]
I think this is not that difficult. I am already working on it. Compilers and scanners are my favorite programming problems.

**Comment by b3b00** [comment]
@huyongde , the solution is near. Could you share you ruse case so I can check if I am on good track ?

**Comment by b3b00** [comment]
@huyongde , @dommmel : [PR 50](https://github.com/valyala/quicktemplate/pull/50) is ready for testing your use cases if you want

**Comment by valyala** [comment]
> {% %} are default delimiters, how to modify the default right and left delimiters, thanks

@huyongde , could you explain the use case where you need to change default delimiters?

**Comment by dalu** [comment]
When you have JS code that uses the same delimiters, then you need to change the default delimiters else you can't use the JS code in the template.

**Comment by b3b00** [comment]
@dalu what is tour JavaScript use case ?

**Comment by dalu** [comment]
@b3b00 what is yours?

**Comment by b3b00** [comment]
@dalu yes Indeed what is your use case... Stupid trench auto correct keyboard
